packages/experimental/Shimmer/src/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from './Shimmer';
-export * from './Shimmer.types';
\ No newline at end of file
+export * from './Shimmer.types';
```

Comment:
Is this file's change worth pushing up? Better to avoid needless edits to files if we don't need them to not muddy the file's history

Reply:
My local copy of prettier or eslint actually won't let me save this file without adding a new line... so yes? 😅

Reply:
Sounds good! If the linter thinks we need it then I think it's fair to have it